subprojects/docs/src/docs/css/manual.css

```diff
@@ -1579,6 +1579,32 @@ div.screenshot {
     margin: 1.5em 0 0;
 }
 
+.docs-navigation li.main-link {
+    font-weight: 500;
+    margin-bottom: 0.2em;
+}
+
+.docs-navigation li.main-link a {
+    display: inline;
+}
+
+.docs-navigation ul.manual-formats {
```

Comment:
This styling is a bit inconsistent with our current navigation. @aweida and I would like to suggest an alternative that allows users to select different versions of Gradle docs from the top navigation.